bin/interpret/funcfile.py

```diff
@@ -140,12 +158,12 @@ def mergeCNfinal(funcDict):
 	#		print 'SMALL SEG NEEDED MERGING'
 
 			#situations where you automatically merge with the previous segment
-			if (i == len(mergePass) - 1) or (j['chrom'] != mergePass[i+1]['chrom']) or (mergePass[i+1]['pass'] == 'no' != mergeSmall[-1]):
+			if (i == len(mergePass) - 1) or (j['chrom'] != mergePass[i+1]['chrom']) or (mergePass[i+1]['pass'] != 'no' and mergeSmall[-1]): #clarified this statement so that last statement couldn't potentially be an automatic merge to the previous segment when no previous segment existed
 		#		print 'auto merge with previous segment'
 				mergeTest = i-1
 				
 			#situations where you automatically merge with the next segment
-			elif (i == 0) or (j['chrom'] != mergeSmall[-1]['chrom']) or (mergeSmall[-1]['pass'] == 'no' != mergePass[i+1]):
```

Comment:
Same as the comment on line 143/161

Would it be saver to flip this and the previous statement? from the perspective of not merging with items that don't exist yet?

Reply:
^See ramblings from 143/161. I actually don't think the merging with items that don't exist should be a problem with either order given the checks for last/first segment being examined.

Reply:
I'll mark this as resolved since the conversation is redundant :)